syntax = "proto3";

package leader;

option cc_generic_services = true;

// The Ping service definition.
service PingService {
  rpc Ping (PingRequest) returns (PingResponse) {}
}

message PingRequest {
  uint32 id = 1;
}

message PingResponse {
  uint32 id = 1;
}
